modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheGroupMetricsImpl.java

```diff
@@ -92,7 +92,7 @@ public CacheGroupMetricsImpl(CacheGroupContext ctx) {
 
         DataStorageConfiguration dsCfg = ctx.shared().kernalContext().config().getDataStorageConfiguration();
 
-        boolean persistentEnabled = CU.isPersistentCache(cacheCfg, dsCfg);
+        boolean persistentEnabled = CU.isPersistentCache(cacheCfg, dsCfg) && !ctx.shared().kernalContext().clientNode();
```

Comment:
Just nip :) It makes sense to swap arguments of `&&` operator.

Reply:
Fixed!